DMTR-91.tex

```diff
@@ -1,564 +1,1334 @@
 
 \section{Personnel}
 \label{sect:personnel}
 
+Following personnel are involved in this test activity:
+
 \begin{itemize}
-\item Test Plan (LVV-P1) Owner: Eric Bellm
-\item Test Run (LVV-R3) Owner: Eric Bellm
-\item LVV-T216 Tester: Eric Bellm
-\item LVV-T217 Tester: Eric Bellm
-\item LVV-T218 Tester: Eric Bellm
-\item Additional Test Personel involved: None 
+\item Test Plan (LVV-P1) owner: Eric Bellm
+\item Test Cycles:
+\begin{itemize}
+  \item LVV-C3 owner: 
+    Eric Bellm
+  \begin{itemize}
+    \item Test case LVV-T216 tester: Eric Bellm
+    \item Test case LVV-T217 tester: Eric Bellm
+    \item Test case LVV-T218 tester: Eric Bellm
+  \end{itemize}
+\end{itemize}
+\item Additional Test Personnel involved: None
 \end{itemize}
 
 \newpage
 
 \section{Overview of the Test Results}
 \label{sect:overview}
 
-\subsection{Summary Table}
+\subsection{Summary}
 \label{sect:summarytable}
 
+\begin{longtable} {p{0.2\textwidth}p{0.2\textwidth}p{0.6\textwidth}}
+\toprule
+  \multicolumn{3}{c}{ Test Cycle {\bf LVV-C3: LDM-503-5: Alert Distribution Validation }} \\\hline
+  {\bf \footnotesize test case} & {\bf \footnotesize status} & {\bf \footnotesize comment} \\\toprule
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T216}{LVV-T216} 
+    & Conditional Pass & Missing git-lfs in steps 2 and 3. Used pre-built Docker image instead.
+Remaining steps pass.
+
+    \\\hline
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T217}{LVV-T217} 
+    & Conditional Pass & Missing git-lfs in step 2. Used docker image instead. Remaining steps
+pass.
+
+    \\\hline
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T218}{LVV-T218} 
+    & Conditional Pass & Missing git-lfs in step 2 and 3. Used docker image instead. Last step
+result deviated sligthly from the expected result for one consumer.
 
-\begin{longtable} {|p{0.2\textwidth}|p{0.2\textwidth}|p{0.6\textwidth}|}\hline
-{\bf TEST CASE ID} & {\bf PASS/FAIL} & {\bf COMMENTS} \\\hline
-LVV-T216 & Conditional Pass & {\textit Missing git-lfs in step 2 and 3. Used docker image instead. Remaining steps pass.} \\\hline
-LVV-T217 & Conditional Pass & {\textit Missing git-lfs in step 2 and 3. Used docker image instead. Remaining steps pass.} \\\hline
-LVV-T218 & Conditional Pass & {\textit Missing git-lfs in step 2 and 3. Used docker image instead. Last step result deviated slightly from the expected result for one consumer.} \\\hline
+    \\\hline
+
+\caption{Test Results Summary}
+\label{table:summary}
 \end{longtable}
 
 \subsection{Overall Assessment}
 \label{sect:overallassessment}
 
-The test campaign is considered in overall: PASS.
-
-Missing elements on the Kubernetes Commons prevented us from building the Docker images in place and using a complete night of unique alerts.
-However, using pre-built Docker images with a smaller, repeated sample of alerts enabled the test to proceed as expected.
+The test campaign is considered in overall:
```

Comment:
“The overall result of the test campaign is: PASS”.

Reply:
I will report this as improvement in docsteady.

Reply:
Sorry for the confusion, this has to be changed in LVV-P1, Overall Assessment field.